### ISSUE TITLE
Report bug/feedback

### ISSUE THREAD

**Issue by Nodonisko** [trezor/trezor-suite#7302]
Screen where user can report bug/send feedback. This will require to have some things solved:
1. Collect logs (we will need them send to us alognside bug report) - #6956
2. Place where we will collect these logs (how is this done in desktop app? will we need a backend for this?)
3. UI/UX defined

```[tasklist]
### Tasks

**Comment by Nodonisko** [comment]
Can you refine this @mnuky ?

**Comment by mnuky** [comment]
> Can you refine this @mnuky ?

Yep, i'm in touch with Tomas from BI and Štěpán from UXR to define the inputs together and Tomáš should help with the technical side. We agreed to do it in the same way as it is in Desktop Suite. We just would like to simplify it in a way that there will be only one form where user can report either bug or feedback.

**Comment by Nodonisko** [comment]
Is this still valid @mnuky ?
